new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -0,0 +1,403 @@
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and domain2.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism.
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // in the dedicated mode, the operator only manages domains in the operator's own namespace
+    domain1Namespace = opNamespace;
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain2Namespace = namespaces.get(1);
+
+    logger.info("Installing and verifying operator");
+    opServiceAccount = opNamespace + "-sa";
+    opHelmParams =
+        new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+  }
+
+  /**
+   * Set "dedicated" to true and set the domainNamespaces to something that does not contain the operator's namespace,
+   * make sure that the domains in the operator's target namespaces do not come up
+   * because when dedicated is set to true, the operator's domainNamespaces value is ignored.
+   * 1) Create CRD.
+   * 2) Install an Operator with namespace=op-ns, domainNamespaces=wls-ns
+   *    and domainNamespaceSelectionStrategy=Dedicated.
+   * 3) Verify the Operator is up and running.
+   * 4) Create WebLogic Domain its namespace=wls-ns.
+   * 5) Verify that domain whose namespace = wls-ns does not come up.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Set dedicated to true and verify that a domains not deployed in operator's namespace doesn't come up")
+  public void testDedicatedModeDiffNamespace() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa",
+        true, 0, opHelmParams, domainNamespaceSelectionStrategy,
+        false, domain2Namespace);
+
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createDomain(domain2Namespace);
+
+    try {
+      Thread.sleep(30 * 1000);
+    } catch (Exception ex) {
+      //ignore
+    }
+
+    verifyDomainNotRunning(domain2Namespace);
+  }
+
+  /**
+   * Set "dedicated" to true, make sure that the domains deployed in the operator's namespace come up.
+   * 1) Create a namespace (e.g. op-wls-ns) that is used b y both Operator and WebLogic domain.
+   * 2) Create CRD.
+   * 3) Install an Operator with namespace=op-wls-ns, domainNamespaces=op-wls-ns
```

Comment:
where are you installing the operator with same namespace for both operator and domain namespaces?

Reply:
changed